CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Cerestats
 
 ## vNext
-- ...
+- Updated logo and favicon.
```

Comment:
In footer you should not change the icon. Please, update.

Reply:
Updated